frontend/src/app/app.module.ts

```diff
@@ -4,14 +4,23 @@ import { RouteReuseStrategy } from '@angular/router';
 import { SplashScreen } from '@ionic-native/splash-screen/ngx';
 import { StatusBar } from '@ionic-native/status-bar/ngx';
 import { IonicModule, IonicRouteStrategy } from '@ionic/angular';
+import { IonicStorageModule } from '@ionic/storage';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 
 @NgModule({
   declarations: [AppComponent],
   entryComponents: [],
-  imports: [BrowserModule, IonicModule.forRoot(), AppRoutingModule],
+  imports: [
+    BrowserModule,
+    IonicModule.forRoot(),
+    IonicStorageModule.forRoot({
+      name: 'digitact',
+      driverOrder: ['indexeddb', 'localstorage'],
```

Comment:
Have you found a way to detect which driver is currently being used? At least in browsers, localstorage has a quota of a few megabytes. If the storage system switches to it, then we will be screwed (since, after storing a few images, the storage will be full.) I guess we should in this case show an error message or so saying that someting is super wrong :D

Reply:
https://ionicframework.com/docs/angular/storage#driver using this we can know the current driver in usage. But this issue exists only ios as far as I tested. We need to somehow try to access content in all the storage sysytem. It can be accessed using indexeddb and localstorage API dorectly just like we do in normal web apps. But this case needs to handled really carefully. Will check.

Reply:
Maybe, the issue you saw was just a temporary glitch? I am just wondering how such a popular library can have such a bug in it :D 

Anyways: I guess as soon as we have integrated SQLite, the issue should be gone

Reply:
May be. SQlite should be a saviour as of now.